src/main/java/mate/academy/spring/model/dto/response/ShoppingCartResponseDto.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.model.dto.response;
+
+import java.util.List;
+
+public class ShoppingCartResponseDto {
+    private Long id;
+    private List<Long> ticketsId;
+    private Long userId;
```

Comment:
As it was mentioned in the checklist, this field is the same as first field "Long id", so we don't need it. In UI we can use first field directly to identify the user

Reply:
Fixed. Thanks a lot!